server/routes/userRouter.ts

```diff
@@ -1,19 +1,35 @@
 import { Router } from 'express';
 import type { Request, Response } from 'express';
-import createUserController from '@/controllers/userController';
+import UserController from '@/controllers/userController';
```

Comment:
Same comment about keeping the factory name as it was to keep it consistent across the backend.